src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -31,8 +33,10 @@ def name(self):
     def setup(self, builder: Builder):
         self.start_date = builder.configuration.time.start
         self.end_date = builder.configuration.time.end
+        self.clock = builder.time.clock()
         self.counter = 0
-        self.output_path = self._build_output_path(builder)
+        self.output_path = self._build_output_root(builder) / "state_table.hdf"
```

Comment:
Is this stuff here because of merge conflicts?

Reply:
Yes.  This is from the observers PR Beatrix submitted and is in here from merging.